liquid-prelude/src/Language/Haskell/Liquid/ProofCombinators.hs

```diff
@@ -153,8 +153,6 @@ infixl 3 ==!
 
 infixl 3 ==.
 
-{-# DEPRECATED (==.) "Use (===) instead" #-}
-
 {-# INLINE (==.) #-} 
 (==.) :: a -> a -> a 
```

Comment:
It could be helpful to have a comment explaining the difference between `==.` and `===` and when to use each.

Reply:
Done!

Reply:
Thank you!